packit_service/worker/parser.py

```diff
@@ -1568,6 +1571,37 @@ def parse_new_hotness_update_event(event) -> Optional[NewHotnessUpdateEvent]:
             release_monitoring_project_id=release_monitoring_project_id,
         )
 
+    @staticmethod
+    def parse_anitya_version_update_event(event) -> Optional[AnityaVersionUpdateEvent]:
+        if "anitya.project.version.update.v2" not in event.get("topic", ""):
+            return None
+
+        # FIXME: Handle Fedora too in case we want to support multiple releases
+        # such as for Go.
```

Comment:
There was an RFE for this on the Slack already…

Reply:
+1, we can create a separate issue for this (e.g.this will imply monitoring settings for the package will not be considered anymore, therefore also changes in documentation etc.) 

we could properly test this for CentOS Stream and once it is tested to be working do the switch for Fedora packages to not break anything for already onboarded Fedora packages